docs/design/access.md

```diff
@@ -202,7 +202,7 @@ Improvements:
 This may provide some defense-in-depth against container escapes. (https://github.com/docker/docker/pull/4572)
 - requires docker to integrate user namespace support, and deciding what
 getpwnam() does for these uids.
-- any features that help users avoid use of privileged containers
+- any features that help users avoid using of privileged containers
```

Comment:
"use of" is correct, but "using" (without the "of") would also be OK.  No change needed